react-version/src/components/game/BoardButtons.js

```diff
@@ -0,0 +1,1221 @@
+import React, { Fragment } from "react";
+
+const BoardButtons = (props) => {
+	return (
+		<Fragment>
```

Comment:
```suggestion
		<>
```
Keep it simple enough ;)

Reply:
Empty tags doesn't work due to .js extend. I'll change it to .jsx in further updates.